doc/Type/X/Bind/Slice.pod6

```diff
@@ -18,7 +18,7 @@ and
     CATCH { default { put .^name, ': ', .Str } };
     # OUTPUT: «X::Bind::Slice: Cannot bind to Hash slice␤»
 
-you get an exception of type X<::Bind::Slice>
+you get an exception of type C<::Bind::Slice>.
```

Comment:
This is out of scope in this PR. Not a bad call, but would have been better in a separate PR.

Reply:
Scratch that, not a PR, but simply commit to main. We should maybe add a test for these.